### ISSUE TITLE
Getting WCAG PDF Check to work

### ISSUE THREAD

**Issue by felixlidl** [veraPDF/veraPDF-wcag-algs#350]
Hello,
I’m not sure if this is the right place to ask this question—if there’s a more appropriate one, please direct me there.

I have a Maven project with Java 21, and I’d like to test some PDFs using your WCAG implementation, but I’m not sure how to do it exactly.

**dependencies in my `pom.xml`**
```
<dependency>
    <groupId>org.verapdf</groupId>
    <artifactId>validation-model-jakarta</artifactId>
    <version>1.28.2</version>
</dependency>

<dependency>
    <groupId>org.verapdf</groupId>
    <artifactId>wcag-validation-jakarta</artifactId>
    <version>1.28.2</version>
</dependency>
````

**My simple Test**
```java
@BeforeAll
    public static void setUpClass() throws UnknownHostException {
        VeraGreenfieldFoundryProvider.initialise();
    }

@Test
    public void testPdfAccessibility() throws Exception {
        try (PDFAParser parser = Foundries.defaultInstance().createParser(new FileInputStream("src/test/resources/Test.pdf"))) {
            PDFAValidator validator = Foundries.defaultInstance().createValidator(PDFAFlavour.WCAG_2_1, false);
            ValidationResult result = validator.validate(parser);
            assertTrue(result.isCompliant());
        } catch (IOException | ValidationException | ModelParsingException | EncryptedPdfException exception) {
            LOG.error(exception.getMessage(), exception);
            fail(exception.getMessage());
        }
    }
```
But I get the following error:
```java
java.util.NoSuchElementException: PDFAFlavour wcag2 is not supported by this directory.

	at org.verapdf.pdfa.validation.profiles.ProfileDirectoryImpl.getValidationProfileByFlavour(ProfileDirectoryImpl.java:106)
	at org.verapdf.pdfa.validation.validators.ValidatorFactory.createValidator(ValidatorFactory.java:72)
	at org.verapdf.pdfa.AbstractFoundry.createValidator(AbstractFoundry.java:88)
	at de.dser.dwh.pdf.test.PdfAccessibilityTest.testPdfAccessibility(PdfAccessibilityTest.java:60)

```

I only found some guidance here: https://docs.verapdf.org/develop/

but I couldn’t find an explanation on how to use PDFAFlavour.WCAG_2_1.

**Comment by bdoubrov** [comment]
Hi @felixlidl WCAG profile is not a part of the official distribution of veraPDF. It has to be treated as a custom profile. 

In general, in order to use custom validation profiles in veraPDF Java SDK, one needs to use a code like this:

```Java
ValidationProfile profile = Profiles.profileFromXml(new FileInputStream("my-custom-profile.xml"));
PDFAValidator validator = Foundries.defaultInstance().createValidator(profile, false);
PDFAParser parser = Foundries.defaultInstance().createParser(new FileInputStream("my-test-document.pdf");
ValidationResult result = validator.validate(parser);
```
